talk/expert/cpp20spaceship.tex

```diff
@@ -1,12 +1,20 @@
 \subsection[spaceship]{The \texttt{<=>} operator}
 
 \begin{frame}[fragile]
-  \frametitlecpp[20]{The three-way comparison operator}
+  \frametitle{The burden of comparison operators}
   \begin{block}{Motivation}
     \begin{itemize}
-    \item There are frequent situations where one needs \mintinline{cpp}{<} operator for a home-made class. Typically, if you want to sort a \mintinline{cpp}{std::vector} of such objects, or use them as a key for e.g. \mintinline{cpp}{std::set} or \mintinline{cpp}{std::map}.
-    \item For completeness, one should also add \mintinline{cpp}{>}, \mintinline{cpp}{>=}, and \mintinline{cpp}{<=}, implemented reusing either \mintinline{cpp}{<} and \mintinline{cpp}{==}, or \mintinline{cpp}{<} and \mintinline{cpp}{>}.
-    \item Those operators should be defined as free functions, optionally friends, so that left and right arguments will be similarly convertible.
+    \item One often needs \mintinline{cpp}{operator<} for a home-made class
+      \begin{itemize}
+      \item e.g. when sorting a \mintinline{cpp}{std::vector} of it
+      \item e.g. when using it as a key for \mintinline{cpp}{std::set} or \mintinline{cpp}{std::map}
+      \end{itemize}
+    \item Many other operators are also desirable for completeness
+      \begin{itemize}
+      \item \mintinline{cpp}{operator>}, \mintinline{cpp}{operator>=}, \mintinline{cpp}{operator<=}, ...
+      \item often implemented reusing e.g. \mintinline{cpp}{operator<} and \mintinline{cpp}{operator==}
```

Comment:
```suggestion
      \item often implemented reusing e.g.\ \mintinline{cpp}{operator<} and \mintinline{cpp}{operator==}
```

Reply:
This looked also fine. So same question : what was the problem ?